helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/CodeTransformations.java

```diff
@@ -47,6 +47,8 @@
   private static final Pattern IMPORT_CODECS_PATTERN = Pattern.compile("import org\\.apache\\.avro\\.message\\.([\\w.]+);");
   private static final Pattern AVROGENERATED_ANNOTATION_PATTERN = Pattern.compile(Pattern.quote("@org.apache.avro.specific.AvroGenerated"));
   private static final Pattern MODEL_DECL_PATTERN = Pattern.compile(Pattern.quote("private static SpecificData MODEL$ = new SpecificData();"));
+  private static final Pattern MODEL_ADD_TYPE_CONVERSION_PATTERN = Pattern.compile("MODEL\\$\\.addLogicalTypeConversion\\(.*\\);");
```

Comment:
using Pattern.quote() would save you all the manual escaping, i think

Reply:
This has a regex in it. So, I can't use `Pattern.quote`